kpops/components/base_components/base_defaults_component.py

```diff
@@ -238,7 +238,7 @@ def defaults_from_yaml(path: Path, key: str) -> dict:
     if value is None:
         return {}
     log.debug(
-        f"\tFound defaults for component type {typer.style(key, bold=True, fg=typer.colors.MAGENTA)} in file:  {path}"
+        f"Found defaults for component type {typer.style(key, bold=True, fg=typer.colors.MAGENTA)}"
```

Comment:
Forgot that we have this 😄 isn't that even better than printing all defaults.yaml in the enrichment path, because this only prints if it actually finds something for a specific component?

Reply:
I would still keep the log in the yaml.py since, in that place, all the pickup YAMLs are logged. For clarity, I added the default paths relative to the cwd here.